src/controllers/ClientController.java

```diff
@@ -38,7 +38,7 @@ public void setFileNameStaff(String fileNameStaff) {
     }
 
     //Inserting Customers name, age, username and password into the database
-    public void insertCustomerToDB(String name, int age, String username, String password, String email, int mobileNumber){
+    public void insertCustomerToDB(String name, int age, String username, String password, String email, String mobileNumber){
```

Comment:
Changed mobileNumber from type int to String

Reply:
uh, we can actually store a 8 digit as an int, because in 32 bits we can store values up to 4 billion (10 digits), and plus changing this would require changes elsewhere too.